### PR TITLE
Sort suppliers

### PR DESCRIPTION
**Pull request by prakash87**

### CODE REVIEW

File: app/main/views/search.py [hunk 1]
```diff
@@ -23,6 +23,7 @@
 
 @main.route('/search/suppliers')
 def supplier_search():
+    sort_by = request.args.get('sort_by', 'asc')  # asc or desc
```
Comment (sarneaud) [inline app/main/views/search.py]: We're only sorting by name now, but the designers have previously talked about adding other sort criteria.  You should make this "name_asc" and "name_desc" (or, better, just use "name" and add a separate parameter for the asc/desc so you don't have to mess with the strings here).

**Comment by matt-sm** [comment]
update/create tests for this?

File: tests/app/views/test_search.py [hunk 2]
```diff
@@ -19,6 +19,13 @@ def find_search_summary(res_data):
     return re.findall(
         r'<span class="search-summary-count">.+</span>[^\n]+', res_data)
 
+@mock.patch('app.main.views.search.DataAPIClient')
+class TestSearchSuppliers(BaseApplicationTest):
+    def test_pagination_links(self, api_client):
```
Comment (sarneaud) [inline tests/app/views/test_search.py]: Is this really testing the pagination links, or just that the page loads?

Comment (prakash87) [inline tests/app/views/test_search.py]: Just the page load for now. Will write some more tests later

File: app/main/views/search.py (hunk 1 above)
Comment (sarneaud) [inline app/main/views/search.py]: This is a sort order, not a sort by, isn't it?

**Pull request merged**